__tests__/app.test.js

```diff
@@ -335,3 +335,62 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("responds with status 200 and an array with correct length", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users).toBeInstanceOf(Array);
+        expect(body.users.length).toBe(4);
+      });
+  });
+  it("responds with status 200 and array with user objects contatining correct properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((user) => {
+          expect(user).toBeInstanceOf(Object);
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+  it("responds with status 200 and stored all users data", () => {

```

Comment:
I appreciate this is thorough but the previous tests take care of this really. And if you were to change the test data it would fail/be a pain to rewrite the assertion - all for no real benefit!